crates/rpc/rpc-eth-api/src/helpers/blocking_task.rs

```diff
@@ -18,6 +20,12 @@ pub trait SpawnBlocking: Clone + Send + Sync + 'static {
     /// Thread pool access in default trait method implementations.
     fn tracing_task_pool(&self) -> &BlockingTaskPool;
 
+    /// See also [Semaphore::acquire_owned](`tokio::sync::Semaphore::acquire_owned`).
+    async fn acquire_owned(&self) -> Result<OwnedSemaphorePermit, AcquireError>;
+
+    /// See also  [Semaphore::acquire_many_owned](`tokio::sync::Semaphore::acquire_many_owned`).
+    async fn acquire_many_owned(&self, n: u32) -> Result<OwnedSemaphorePermit, AcquireError>;
```

Comment:
we can also return impl Future here and don't need async_trait, see `spawn_blocking_io` for ref

Reply:
Done